src/condor_daemon_core.V6/condor_timer_manager.h

```diff
@@ -122,7 +121,11 @@ class TimerManager
                   time_t deltawhen,
                   TimerHandlercpp handler,
                   const char * event_descrip,
-                  time_t       period          =  0);
+                  time_t       period          =  0) {
+		return NewTimer(deltawhen, period,
+				[s, handler](int id) { return (s->*handler)(id);},
```

Comment:
I don't think it matters in this case, but we should talk about idiom(s) for / idiomatic lambdas: I think I prefer implicit captures but explicitly by-value semantics over the explicit captures with implicit by-reference semantics here.